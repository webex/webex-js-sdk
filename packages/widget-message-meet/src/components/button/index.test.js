import React from 'react';
import renderer from 'react-test-renderer';

import Button from '.';

describe(`Button component`, () => {
  const onClick = jest.fn();
  const component = renderer.create(
    <Button
      iconType="delete"
      onClick={onClick}
    />
  );

  it(`renders properly`, () => {
    expect(component).toMatchSnapshot();
  });
});
