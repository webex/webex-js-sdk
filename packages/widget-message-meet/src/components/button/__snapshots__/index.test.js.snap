exports[`Button component renders properly 1`] = `
<button
  className="button"
  onClick={[Function]}
  onKeyUp={[Function]}>
  <div
    className="icon"
    title={undefined} />
</button>
`;
